Documentation/flexvolume.md

```diff
@@ -81,6 +81,10 @@ If you want to use Rook with an already provisioned Tectonic cluster, please ref
 
 Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
 
+### OpenStack Magnum
+OpenStack Magnum using Atomic uses a non-standard FlexVolume plugin directory:  `/var/lib/kubelet/volumeplugins`
```

Comment:
@emonty Did you get this working without setting the `FLEXVOLUME_DIR_PATH` in the operator as described in the section below? I would think you would need that so the rook flex driver would be copied to the correct place. If it's needed, can you point them to it? Similar to the "Continue with ..." paragraphs in other sections.

Reply:
Ah - yes, setting the ```FLEXVOLUME_DIR_PATH``` is required. Updating now.